src/select-list-view.js

```diff
@@ -37,6 +37,14 @@ module.exports = class SelectListView {
     this.refs.queryEditor.setText('')
   }
 
+  setQueryText (value, isSelect = false) {
```

Comment:
I believe it helps clarify the purpose of the boolean to people that haven't written the code (or, after some months, to the person who wrote the code). Also, in case we add new parameters, they can just be added to the object as opposed to being passed as arguments ( which would make the API intent even more obscure), e.g.

```js
selectList.setQueryText('hello', true, 1, 3)
// vs.
selectList.setQueryText(
  'hello', 
  {selectText: true, selectionStart: 1, selectionEnd: 3}
)
```